### ISSUE TITLE
Getting attacked notification not getting shown properly

### ISSUE THREAD

**Issue by derfuhrer2007** [jsettlers/settlers-remake#404]
For some reason player0 (me) is very rarely getting notified of attacks, usually you get the notification of other players to know that you are attacked. I think this is done differently, for some reason. Here is one example, and this is the rare times I actually got a player0 notification
[2016-05-03_20-23-10_mountain_lake (2).zip](https://github.com/jsettlers/settlers-remake/files/248358/2016-05-03_20-23-10_mountain_lake.2.zip)

![screenshot 8](https://cloud.githubusercontent.com/assets/18621498/15008267/b323d698-11fc-11e6-9884-4f7b7962fa06.png)

**Comment by andreas-eberle** [comment]
in fact, a player should never see the attacked messages of other players. We need to filter out the messages that are not relevant for the current player.

**Comment by derfuhrer2007** [comment]
This is that the messages for player0 were very few and far between, though I was in the middle of most of the fighting

**Comment by andreas-eberle** [comment]
Yes, I understood that. But I think the main problem, that is also causing your issue, is that you see the messages of others.

@JKatzwinkel: You did some work on the message system already. Do you want to take a look if you can fix this?

**Comment by JKatzwinkel** [comment]
Only thing that seems a bit off here are those 'player 0'-sent notifications. Will look into it.

@derfuhrer2007 what kind of fucked-up username is this??

**Comment by derfuhrer2007** [comment]
The gaming nickname my lil bro picked up 9 years back. Why?

**Comment by michaelzangl** [comment]
"Der Führer" is sort of a German military rank that only one person in history had. That person is not popular and there are many German people participating in this project.
